backend/src/tasks/peCybersixgill.ts

```diff
@@ -0,0 +1,24 @@
+import { CommandOptions } from './ecs-client';
+import { spawnSync } from 'child_process';
+
+// Call the sync_dnstwist_pe.py script that fetches dnstwist results, checks
+// IPs using the blocklist.de api, then updates the PE db instance
+
+export const handler = async (commandOptions: CommandOptions) => {
+  const { organizationId, organizationName } = commandOptions;
+  const child = spawnSync(
+    'python3',
+    ['/app/worker/pe_scripts/sixgill/run_cybersixgill.py'],
+    {
+      stdio: 'pipe',
+      encoding: 'utf-8',
+      env: {
+        ...process.env,
```

Comment:
Let's create a common utils function called `getPeEnv()` which sends only the P&E env variables required to the python scripts (such as PE_DB_USERNAME, PE_DB_PASSWORD, etc.).

Reply:
Addressed here https://github.com/cisagov/crossfeed/pull/1413/commits/2223c7dcc0b9db1f92e5605a3608b4ee38d0ae5f